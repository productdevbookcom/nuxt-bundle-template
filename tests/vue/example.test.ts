import { describe, expect, test } from 'vitest'
import { mount } from '@vue/test-utils'
import Button from '~/vue/src/App.vue'

describe('button', () => {
  test('alert render', () => {
    const wrapper = mount(Button)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
