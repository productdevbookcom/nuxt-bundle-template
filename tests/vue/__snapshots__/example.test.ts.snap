// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`button > alert render 1`] = `"<div><button>Test button</button></div>"`;

exports[`button > disabled 1`] = `"<div disabled=\\"true\\"><button>Test button</button></div>"`;

exports[`button > loading 1`] = `"<div loading=\\"true\\"><button>Test button</button></div>"`;
